templates/registration/login.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html' %}
+{% load crispy_forms_tags %}
+
+{% block title %}Login{% endblock %}
+
+{% block content %}
+{% include 'home.html' %}
+<div class="container">
+    <div class="col-md-4 m-auto">
+        <div class="text-center">
+            <h1 class="mt-5 mb-4 ">Login</h1>
+        </div>
+        <form class="mb-2"  method="post">
+            {% csrf_token %}
+            <table>
+                {{form |crispy}}
```

Comment:
why we put `crispy` filter here?

Reply:
it's just for Django customs form markup to bootstrap4 markup. 

Ref: https://pypi.org/project/django-crispy-forms/